airflow/providers/openlineage/extractors/base.py

```diff
@@ -35,6 +37,15 @@ class OperatorLineage:
     run_facets: dict[str, BaseFacet] = Factory(dict)
     job_facets: dict[str, BaseFacet] = Factory(dict)
 
+    @staticmethod
+    def merge(base: OperatorLineage, other: OperatorLineage) -> OperatorLineage:
+        return OperatorLineage(
+            inputs=base.inputs + other.inputs,
+            outputs=base.outputs + other.outputs,
+            run_facets=merge_dicts(base.run_facets, other.run_facets),
```

Comment:
I would indicate here that in the future we might need a more sophisticated solution

Reply:
Added description.

Reply:
Added description.